### PR TITLE
Add docs for `schedule` and `unschedule` command

### LINKED ISSUE #16: Update UG - `schedule` command

### PR DESCRIPTION
**Pull request by Dethada**
closes #16 and #17

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -144,6 +144,44 @@ Examples:
...
 
+### Scheduling a meeting : `schedule`
+
+Schedules a meeting with the specified person.
+
+Format: `schedule INDEX s/START_DATETIME e/END_DATETIME`
+
+* The index refers to the index number shown in the displayed person list.
+* The index **must be a positive integer** 1, 2, 3, …​
+* START_DATETIME must be after the current datetime
```
Comment (Javiery3889) [inline docs/UserGuide.md:155]:
```suggestion
* START_DATETIME must be after the system's current datetime.
```
Just a small nit, perhaps specifying system datetime would be better here?

File: docs/UserGuide.md [hunk 2]
```diff
@@ -144,6 +144,44 @@ Examples:
 * `list` followed by `delete 2` deletes the 2nd person in the address book.
 * `find Betsy` followed by `delete 1` deletes the 1st person in the results of the `find` command.
 
+### Scheduling a meeting : `schedule`
+
+Schedules a meeting with the specified person.
+
+Format: `schedule INDEX s/START_DATETIME e/END_DATETIME`
```
Comment (KohGuanZeh) [inline docs/UserGuide.md:151]: Noticed that prefix is the same as `EMAIL` prefix. As of now there is no commands that `END_DATETIME` is shared with `EMAIL`. Is it fine to use the same prefix, or is it better to change the prefix now? @Dethada @Javiery3889 @YuZizhen @Arixeyeion

**Review by KohGuanZeh — COMMENTED** [review]
Only have concerns with prefix for `END_DATETIME`. Apart from that LGTM.

File: docs/UserGuide.md (hunk 2 above)
Comment (Javiery3889) [reply docs/UserGuide.md:151]: Perhaps it will be better to use the same prefix, as each command will have its own set of prefixes

**Review by Javiery3889 — APPROVED** [review]
LGTM 👍

**Review by KohGuanZeh — APPROVED** [review]

**Pull request merged**